src/popover/popover.directive.ts

```diff
@@ -33,6 +41,10 @@ export class PopoverDirective implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() public container: string;
+  /**
+   * Specifies if popover should be closed after clicking outside
+   */
+  @Input() public popoverCloseOnClickOutside: boolean;
```

Comment:
Not good... In my case there is a big component with several injected sevices and I wouldn't want there another just for configuring a popover. Can't we leave developers the free choice how they want to use it?